docs/features/techdocs/using-cloud-storage.md

```diff
@@ -181,14 +181,35 @@ techdocs:
 ```
 
 Refer to the
-[official AWS documentation for obtaining the credentials](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/getting-your-credentials.html).
+[official AWS documentation for obtaining the credentials](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/getting-your-credentials.html).
 
 Note: If you are using Amazon EC2 instance to deploy Backstage, you do not need
 to obtain the access keys separately. They can be made available in the
 environment automatically by defining appropriate IAM role with access to the
 bucket. Read more
 [here](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html#use-roles).
 
+**3c. Authentication using an assumed role** Users with multiple AWS accounts
+may want to use a role for S3 storage that is in a different AWS account. Using
+the `roleArn` parameter as seen below, you can instruct the TechDocs publisher
+to assume a role before accessing S3.
+
+```yaml
+techdocs:
+  publisher:
+    type: 'awsS3'
+    awsS3:
+      bucketName: 'name-of-techdocs-storage-bucket'
+      region:
+        $env: AWS_REGION
+      credentials:
+        roleArn: arn:aws:iam::123456789012:role/my-backstage-role
+```
+
+Note: Assuming a role requires that primary credentials are already configured
+at `AWS.config.credentials`. Read more about assuming roles
```

Comment:
Perhaps you meant this.

```suggestion
at `techdocs.publisher.awsS3.config.credentials` as shown in 3b above. Read more about assuming roles
```

Reply:
I did not - the AWS assume role credentials provider uses the default existing credentials to assume a role, which are either implicitly or explictly configured at AWS.config.credentials